Docs/source/usage/faq.rst

```diff
@@ -50,6 +50,23 @@ Here are a few practical items to assist in designing boosted frame simulations:
 
 An in-depth discussion of the boosted frame is provided in the :ref:`moving window and optimal Lorentz boosted frame <theory-boostedframe>` section.
 
+What about Back-transformed diagnostics (BTD)?
+----------------------------------------------
+
+.. figure:: BTD_features.png
+   :alt: [fig:BTD_features] Minkowski diagram indicating several features of the back-transformed diagnostic (BTD). The diagram explains why the BTD may not start at :math:`t=0`, why the BTD grid-spacing and snapshot length don't correspond to the grid spacing and length in the input script, and when the snapshot is completed.
+
+   [fig:BTD_features] Minkowski diagram indicating several features of the back-transformed diagnostic (BTD). The diagram explains why the BTD may not start at :math:`t=0`, why the BTD grid-spacing and snapshot length don't correspond to the grid spacing and length in the input script, and when the snapshot is completed.
+
+
+Several BTD quantities differ slightly from the lab frame domain described in the input deck.
+In the following discussion, we will use a subscript input (e.g. :math:`\Delta z_{\rm input}`) to denote properties of the lab frame domain.
+
+
+- The first back-transformed diagnostic (BTD) snapshot may not occur at :math:`t=0`. Rather, it occurs at :math:`t_0=\frac{z_{max}}c \beta(1+\beta)\gamma^2`. This is the first time when the boosted frame can complete the snapshot.
```

Comment:
Should `t=0` here be `t'=0`, with `t'` being the time in the boosted frame? Same question for `t_0`. Not sure, maybe I'm confused by what we mean when we say that a BTD snapshot "occurs".

Reply:
Every BTD snapshot is meant to be a spatial snapshot of the simulation at exactly one time in the lab frame.  Here I'm trying to say that even though we might expect the first BTD snapshot to be a snapshot of the simulation at time `t=0`, i.e. should be the initial data, it may not.  It is rather a snapshot at time `t=t_0` where `t_0` depends on the boost and on the right end of the computational domain `z_max`